handbook_reviewer_guidelines.md

```diff
@@ -0,0 +1,25 @@
+This document contains the guidelines for anyone reviewing content for the Operate First Community Handbook.
+
+== Styling content
+
+
+== Criteria for each column to progress to the next column:
+=== To enter ‘Proposed content’
+- [ ] An issue or PR is made and assigned to the ‘Community Handbook’ project
+- [ ] To enter “Content in progress” (decider = reviewer)
+- [ ] Full idea
+- [ ] For issues, template is completely filled out
+- [ ] Proposed idea is understood by a reviewer
+- [ ] Reviewer accepts the idea and confirms it should be worked on (communication done through comments)
+- [ ] For issues, at least one person is assigned to work on it
+=== To enter “Review in progress” (decider = writer)
+- [ ] Writer (assignee) has worked on content / solution
+- [ ] Writer feels the content is ready for review
+=== To enter “Reviewer approved” (decider = reviewer)
+- [ ] Reviewer has fully reviewed the content, which may involve a series of suggestions and improvements
+- [ ] Content meets the [style guidelines](style_guide.md)
+  - Complete has a lower standard than “complete” for user-facing documentation, as per the Community Handbook style guide.
+- [ ] Reviewer feels the content is complete/accurate and approves it
+=== To enter ‘Done’ (decider = editor/reviewer & writer)
```

Comment:
Should we delete editor here in "decider = editor/reviewer & writer" - I believe we removed editor from the other lines to make things more clear

Reply:
Sure enough, fixed, thanks!